tests/template/code_variable_test.py

```diff
@@ -44,6 +44,9 @@ def test_cache():
                         ),
                     ],
                     commands=[],
+                    bunch=PKDict(
```

Comment:
We should remove the changes to this test because the lattice ModelIterator only looks inside elements and commands values, so the top-level bunch expression would never be visited.